modules/core/src/main/scala/org/scalasteward/core/util/wart.scala

```diff
@@ -0,0 +1,24 @@
+/*
+ * Copyright 2018-2020 Scala Steward contributors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.scalasteward.core.util
+
+object wart {
+  @SuppressWarnings(Array("org.wartremover.warts.Equals"))
```

Comment:
We normally deal with this wart by adding a `cats.Eq` instance for the types we want to compare and then using `import cats.implicits._` at the site where the comparison happens so that we can use Cats' `===` or `=!=` syntax. See for example [here](https://github.com/scala-steward-org/scala-steward/blob/00fef541a845f232fb30154b372742c3852cde3d/modules/core/src/main/scala/org/scalasteward/core/git/GitAlg.scala#L169) and [here](https://github.com/scala-steward-org/scala-steward/blob/00fef541a845f232fb30154b372742c3852cde3d/modules/core/src/main/scala/org/scalasteward/core/git/Sha1.scala#L42).

Reply:
Thanks, changed